src/platforms/php/guides/symfony/performance/pm-integrations.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Performance Integrations
+description: "Learn about the the integrations Sentry provides for performance monitoring and how to configure them."
+sidebar_order: 500
+---
+
+The Symfony SDK offers integrations that support distributed tracing, including database queries, Twig template rendering, and cache pools. These integrations are disabled by default to avoid adding unexpected overhead to your services.
+
+To use them, you need to use the latest version of the SDK and add the specific integration as illustrated below:
```

Comment:
These below are the default settings, if each of those components are present.

Reply:
As long as the vendors (DBAL, Cache and Twig) are installed the `enabled` key will be set to `true`, so we may rephrase a bit this paragraph to tell users that it's likely that this config is the same of their app, but they should adjust it to their needs if they want to disable specific integrations.

Reply:
I've added this to the intro.